devops/pipeline/binary_package.yaml

```diff
@@ -16,84 +16,84 @@ parameters:
       - variantName: ubuntu20.04
         friendlyVariantName: ubuntu2004
         architecture: amd64_debug
-        containerPath: $(CONTAINER_REGISTRY)/ubuntu20.04-dev-amd64:latest
+        containerPath: $(CONTAINER_REGISTRY)/ubuntu20.04-dev-amd64:270554
```

Comment:
>270554 [](http://example.com/codeflow?start=67&length=6)

Can some global variable like this deleted 'latest' be used and be set once to this hard coded value 270554? Instead of setting the hard-coded values everywhere